File: src/main/java/social/alone/server/location/LocationDto.java
package social.alone.server.location;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@AllArgsConstructor
@NoArgsConstructor
public class LocationDto {

  public String address;
  public String name;
  public Double longitude;
  public Double latitude;
  public String placeUrl;

  public Location buildLocation() {
    return new Location(address, name, longitude, latitude, placeUrl);
  }
}

File: src/main/kotlin/social/alone/server/location/LocationDto.kt
package social.alone.server.location


import lombok.AllArgsConstructor
import lombok.Getter
import lombok.NoArgsConstructor

@Getter
@AllArgsConstructor
@NoArgsConstructor
data class LocationDto(
        var address: String = "",
        var name: String = "",
        var longitude: Double,
        var latitude: Double,
        var placeUrl: String=""
) {


    fun buildLocation(): Location {
        return Location(address, name, longitude, latitude, placeUrl)
    }
}